doc/TBG_macros.cfg

```diff
@@ -96,17 +96,18 @@ TBG_movingWindow="-m"
 # For a full description, see the plugins section in the online wiki.
 #--<species>_radiation.period 	Radiation is calculated every .period steps. Currently 0 or 1
 #--<species>_radiation.dump 	Period, after which the calculated radiation data should be dumped to the file system
-#--<species>_radiation.lastRadiation 	If set to 1, the spectra summed between the last and the current dump-time-step are stored
+#--<species>_radiation.lastRadiation 	If set the spectra summed between the last and the current dump-time-step are stored
 #--<species>_radiation.folderLastRad 	Folder in which the summed spectra are stored
-#--<species>_radiation.totalRadiation 	Set to 1 to store spectra summed from simulation start till current time step
+#--<species>_radiation.totalRadiation 	Set to to store spectra summed from simulation start till current time step
 #--<species>_radiation.folderTotalRad 	Folder in which total radiation spectra are stored
 #--<species>_radiation.start 	Time step to start calculating the radition
 #--<species>_radiation.end 	Time step to stop calculating the radiation
 #--<species>_radiation.omegaList 	If spectrum frequencies are taken from a file, this gives the path to this list
-#--<species>_radiation.radPerGPU 	If set to 1, each GPU stores its own spectra without summing the entire simulation area
+#--<species>_radiation.radPerGPU 	If set, each GPU stores its own spectra without summing the entire simulation area
 #--<species>_radiation.folderRadPerGPU 	Folder where the GPU specific spectras are stored
-TBG_radiation="--<species>_radiation.period 1 --<species>_radiation.dump 2 --<species>_radiation.totalRadiation 1 \
-               --<species>_radiation.lastRadiation 0 --<species>_radiation.start 2800 --<species>_radiation.end 3000"
+#--e_<species>_radiation.compression    If set, the hdf5 output will be compressed.
+TBG_radiation="--<species>_radiation.period 1 --<species>_radiation.dump 2 --<species>_radiation.totalRadiation \
+               --<species>_radiation.lastRadiation --<species>_radiation.start 2800 --<species>_radiation.end 3000"
```

Comment:
example changes meaning for `lastRadiation` but I guess you did that intentionally.